backend/webserver/tests.py

```diff
@@ -217,3 +224,56 @@ def test_logout(self):
         """Always logs out a request"""
         response = self.client.post("/logout/")
         self.assertEqual(status.HTTP_200_OK, response.status_code)
+
+class PostTestCase(APITestCase):
+    
+    def test_get(self):
```

Comment:
We need a test for GET that verifies the full format as listed here - https://github.com/zarifmahfuz/project-socialdistribution/blob/master/project.org#post. In other words, the test needs to verify that all fields mentioned in the specified format are present in the response